test/e2e/tools/tools_test.go

```diff
@@ -48,5 +63,77 @@ var _ = Describe("[tools] Auxiliary tools", func() {
 			_, ok := platform.ParsePlatform(text)
 			Expect(ok).To(BeTrue(), "cannot recognize detected platform: %s", text)
 		})
+
+		It("[api][mkginkgolabelfilter] should expose correct active features and create filter", Label("api", "mkginkgolabelfilter"), func(ctx context.Context) {
```

Comment:
note the `mkginkgolabelfilter` tests can be added later. So the change I'm suggesting here is meant to _reduce_ the scope of this PR rather than expand it further

Reply:
thanks for your comment, I did thought about splitting the API check and the tool one but I would need to duplicate the API check so combined that as one test. Your suggestion sounds better though so ill adopt it. thanks!

Reply:
regarding the first point,  if we want to go with local binary we’ll miss the version check so either we drop its check in the validate method or set it manually to match the expected and use validate as it is, which is better than completely dropping it as we are consuming binaries of the same code version, I'll adjust this anyway

Reply:
I'm not against running against the cluster if it is helpful. Let's try to require cluster in the tools lane _if we can_, but if cluster access is needed, so be it.